test/Eryph.ConfigModel.Core.Validation.Tests/StorageIdentifierTests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Eryph.ConfigModel.Core.Validation.Tests;
+
+public class StorageIdentifierTests
+{
+    [Fact]
+    public void Validate_AutoGeneratedId_ReturnsSuccess()
+    {
+        var result = StorageIdentifier.NewValidation("BE8BUPERXP1C");
+
+        result.Should().BeSuccess()
```

Comment:
@fw2568 We decided to normalize names to lower case but the auto-generated storage identifiers are upper case. Should we change this for consistency?

Reply:
yes